evm/vm/base.py

```diff
@@ -231,7 +232,7 @@ def create_unsigned_transaction(self, *args, **kwargs):
 
     def validate_transaction(self, transaction):
         """
-        Perform evm-aware validation checks on the transaction.
+        Perform chain-aware validation checks on the transaction.
```

Comment:
No, this is more chain aware.  If you look at https://github.com/pipermerriam/py-evm/blob/master/evm/vm/flavors/frontier/validation.py#L6 it accesses block data which is a chain level thing.